spec/controllers/api/v0/efile_controller_spec.rb

```diff
@@ -1,14 +1,26 @@
 require "rails_helper"
 
 describe Api::V0::EfileController, type: :controller do
+  let(:api_client_name) { "api_client" }
+  let(:api_request_id) { "fake-uuid" }
   let(:mef_credentials) { {mef_env: "test", app_sys_id: "foo", etin: "bar", cert_base64: "baz"} }
 
   before do
     allow_any_instance_of(described_class).to receive(:verify_client_name_and_signature).and_return(true)
-    allow_any_instance_of(described_class).to receive(:get_api_client_mef_credentials).and_return(mef_credentials)
+    allow_any_instance_of(described_class).to receive(:api_client_name).and_return(api_client_name)
+    allow_any_instance_of(described_class).to receive(:api_request_id).and_return(api_request_id)
+    allow(MefService).to receive(:get_mef_credentials).and_return(mef_credentials)
+  end
+
+  after do
+    stub_request(:post, webhook_url).with(body: expected_webhook_request_body)
+    clear_performed_jobs
+    perform_enqueued_jobs(only: WebhookCallbackJob)
```

Comment:
why do you perform this job in the after block? it appears the individual tests perform the enqueued job for a each test?

Reply:
yep, enqueued jobs are not performed in tests without calling `perform_enqueued_jobs`. the tests call it to perform the job enqueued by the controller (e.g. Submit, Acks, SubmissionsStatus), and then because they all enqueue a webhook callback job, I just put the test code to perform that job in the after hook to not duplicate it for each test